src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/edit-polling-source.service.ts

```diff
@@ -60,15 +60,18 @@ export class EditPollingSourceService {
         }
     }
 
-    public getSetPollingSourceAsYaml(
+    public getEventAsYaml(
```

Comment:
If it is now a general algorithm, it should not be placed in a file about `SetPollingSource`

Reply:
Move to base class